src/components/AccountModule/WalletConnectionInfo.js

```diff
@@ -118,8 +118,6 @@ function WalletConnectionInfo({
                 display: flex;
                 align-items: center;
                 justify-self: flex-end;
-                padding: ${0.5 * GU}px;
-                &:active {
                   background: ${theme.surfacePressed};
                 }
```

Comment:
We probably want to remove this dangling end `}` and re-indent `background`.

Was this meant to be changed though?

Reply:
Hmm yeah, I don't think this was meant to be changed? You can see the shaded border at all times.

<img width="193" alt="Screen Shot 2020-02-12 at 11 47 00 PM" src="https://user-images.githubusercontent.com/4166642/74384629-fe78ad00-4df1-11ea-81aa-a80367685f8a.png">

Reverted in df5e9bb.